### PR TITLE
Check why/if ua.tests.doc require jdt/pde

### PR DESCRIPTION
**Pull request by laeubi**

### CODE REVIEW

**Comment by laeubi** [comment]
@akurtakov @vogella anyone knows *what* actually these tests "test" and why we have a requirement on JDT/PDE here?

**Comment by laeubi** [comment]
Only failing test is `org.eclipse.ua.tests.doc.internal.linkchecker.ApiDocTest` that seem to test some converns from JDT/PDE I think we have two options here:

1. remove references and don't care about PDE/JDT
2. PDE/JDT can copy the test and check their parts directly in the repo

the test iself seems self-contained enough for this.

**Comment by akurtakov** [comment]
I would be fine with such approach for testing.

File: org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java [hunk 1]
```diff
@@ -103,8 +103,9 @@ public void endElement(String uri, String localName, String qName) {
 
 	private static final String[] TOCS = {
 			"org.eclipse.platform.doc.isv",
-			"org.eclipse.jdt.doc.isv",
-			"org.eclipse.pde.doc.user",
+			// FIXME why do we test JDT + PDE here ?!?
```
Comment (iloveeclipse) [inline org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java:106]: The ApiDocTest that run **on jenkins** is meaningless. It is supposed to run in full SDK build and test that all the API we ship in SDK doesn't have broken references - and that needs both JDT / PDE.

So if you want fix jenkins build, please **conditionally** disable it for jenkins.

**Review by iloveeclipse — CHANGES_REQUESTED** [review]

**Pull request closed without merging**